Multisig/Logic/Models/TokenBalance.swift

```diff
@@ -76,7 +76,7 @@ extension TokenBalance {
     static func displayCurrency(from serverValue: String, code: String) -> String {
         let number = serverCurrencyFormatter.number(from: serverValue) ?? 0
         let formatter = TokenFormatter()
-        let amount = Int256(number.doubleValue) * 100
+        let amount = Int256(number.doubleValue * 100)
```

Comment:
Does this work for more or less than 2 minor units? 
There are currencies that have 0 or 3 minor units of currency.

https://en.wikipedia.org/wiki/ISO_4217#Minor_units_of_currency

Reply:
BTW: We should never use floating point types for Fiat values. Always integer including minor units. But that is something for another PR :-)

Reply:
apparently there is no out of the box support for ISO4217 in ios. NumberFormatter returns 2 decimal digits even for codes that should have 3 or 4. In fiat list that we get from cgw that would be CLF and IQD
Issue opened: https://app.zenhub.com/workspaces/mobile-development-board-60cc78740f488d000e996335/issues/gnosis/safe-ios/1837